charts/fluid-dataprocess/common/templates/job.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ printf "%s-job" .Release.Name }}
```

Comment:
"%s-job" -> "%s-process" ?

Reply:
`.Release.Name` already  contains `processor`. For example a DataProcess CR named `myprocess` will have a job named `myprocess-processor-job`

Reply:
I see